core/src/main/scala/cats/std/either.scala

```diff
@@ -23,7 +23,7 @@ trait EitherInstances extends EitherInstances1 {
         }
     }
 
-  implicit def eitherInstances[A]: Monad[Either[A, ?]] with Traverse[Either[A, ?]] =
+  implicit def catsMonadForEither[A]: Monad[Either[A, ?]] with Traverse[Either[A, ?]] =
```

Comment:
@larsrh do you have any thoughts on this? You did a really good job of maintaining binary compatibility in scalaz, and I'd imagine this sort of thing came up.